components/MdInput.test.js

```diff
@@ -40,8 +41,61 @@ describe('MdInput Component', () => {
 
     const textbox = queryByTestId('textbox')
     const markdown = queryByTestId('markdown')
-    expect(textbox).toBeNull()
+    expect(textbox.classList.contains('d-none')).toBe(true)
```

Comment:
You could use `jest-dom` here to make this simpler, see [`toHaveClass(className)`](https://github.com/testing-library/jest-dom#tohaveclass). Just have to import it for this test.

It helps to make the tests a little bit more cleaner imo.

```suggestion
    expect(textbox).toHaveClass('d-none')
```